orc8r/cloud/go/services/state/state/main.go

```diff
@@ -105,7 +105,7 @@ func newIndexerManagerServicer(cfg *config.ConfigMap, db *sql.DB, store blobstor
 	reindexer := reindex.NewReindexer(queue, reindex.NewStore(store))
 	servicer := servicers.NewIndexerManagerServicer(reindexer, autoReindex)
 
-	if autoReindex && storage.SQLDriver != sqorc.PostgresDriver {
+	if autoReindex && storage.GetDatabaseSource() != sqorc.PostgresDriver {
```

Comment:
Here and elsewhere: some of these are mixing `SQLDriver` and `DatabaseSource`

Reply:
good catch went a little overboard with find/replace